06_iito/index.tex

```diff
@@ -1 +1,34 @@
+
+\section{Inter-Island Communication}  
+\label{sec:IITO:inter_island_communication}
+
+Islands being able to communicate independently from the main governing body allows for more complex island behaviour. With the designed system, islands can:
+
+\begin{itemize}
+    \item Form a group with other islands for collective foraging.
+    \item Decide where their foraging group will forage.
+    \item Inform other islands of the amount they intend to to donate to the common pool.
+    \item Share voting history with other islands.
+    \item Share tax amount history with other islands.
+    \item Share the amount of resources they have with other islands.
+\end{itemize}
+
+\textbf{Team Foraging: }Allowing islands to decide between each other on the groups that they will forage in and where that group will forage means that they can work with islands that they trust while avoiding islands that have misbehaved or broken rules in the past. Islands may also want to avoid foraging with other islands that have contributed less than expected in the past if they suspect that they may have been deliberately freeloading.
+
+\textbf{Common Pool Donations: }Allowing islands to broadcast how much they plan to donate to the common pool lets them declare to the other islands if they plan to donate more than the specified tax amount put forward by the president as a form of virtue signalling. This may help an island redeem itself if it had previously lost trust. An island can lie about the amount it will donate if it is trying to maliciously gain favour but this can be caught by the Judge if they choose to inspect it, which may result in punishment.
+
+\textbf{Voting History: }Islands can request voting history from other islands or provide their own unprompted. This lets islands try to verify that the Speaker has been honest when counting votes but the islands can lie in the history that they provide, meaning islands may want to only take heed of information from islands they already trust. 
+
+\textbf{Tax History: }In a similar sense, islands can request taxation history (how much the islands were told to give in tax by the President) from other islands but an island may choose to not provide this history or be dishonest about it. If the islands are honest, this allows for more transparency and lets the islands better judge if the President is performing their role correctly.
+
+\textbf{Current Resources: }Islands are also able to share the value of their current resources. This may assist islands when making decisions regarding gifts (Section~\ref{sec:IITO:gifting}). For example, if an island requests a gift because they are low on resources, the island receiving the request may want to ask how many resources the requesting island has. This can also give an island an indication of the overall success of the archipelago. Similarly to intended common pool donations and voting and tax history sharing, islands can lie or withhold this information.
```

Comment:
suggested re-writing:

Islands are also able to share the value of the resources that they currently have in their private pool. This information may guide islands when making decisions regarding gifts (Section~\ref{sec:IITO:gifting}). For example, if an island requests a gift because they are low on resources, the island receiving the request may want to ask how many resources the requesting island has. This can also give an island an indication of the overall level of richness in the archipelago. Similarly to sharing the intended common pool donations along with voting and tax history, the islands can decide not to report or lie about their current amount of resources.